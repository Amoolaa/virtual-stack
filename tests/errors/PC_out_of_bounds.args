"tests/errors/PC_out_of_bounds.x2017"
