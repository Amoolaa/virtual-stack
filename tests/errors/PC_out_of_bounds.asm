FUNC LABEL 0
    MOV STK A VAL 9
    MOV REG 0 STK A
    REF STK B STK A
    MOV REG 7 VAL 10
    PRINT PTR B
    CAL VAL 6
    RET
FUNC LABEL 6
    PRINT VAL 9
    PRINT VAL 5
    MOV STK A VAL 1
    PRINT STK A
    RET
