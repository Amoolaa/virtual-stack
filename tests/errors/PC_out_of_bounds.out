Program counter out of bounds. Instruction 10 does not exist for function 0.
